css/properties.json

```diff
@@ -4220,6 +4220,50 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/contain"
   },
+  "contain-intrinsic-size": {
+    "syntax": "[ none | <length> | auto <length> ]{1,2}",
```

Comment:
This is copied from spec: https://w3c.github.io/csswg-drafts/css-sizing-4/#propdef-contain-intrinsic-size